Library/Formula/curl.rb

```diff
@@ -47,7 +51,7 @@ def install
     if build.with?("libressl") && build.with?("openssl")
       ohai <<-EOS.undent
       --with-openssl and --with-libressl are both specified and
-      curl can only use one at a time; proceeding with openssl.
+      curl can only use one at a time; proceeding with libressl.
```

Comment:
Sadly, this could cause problems. The last three versions of ` libressl ` have had some... problems, working with the generated ca certs, which is causing a few connections to fail without any particularly informative message. Spent a good 7 hours chasing it around earlier, which was fun. We need to default to remain ` openssl `, despite my broader desires and admiration of the ` libressl ` codebase.